### ISSUE TITLE
Update Terraform AWS GitHub runner (Q2 2022)

### ISSUE THREAD

**Issue by stephanosio** [zephyrproject-rtos/infrastructure#23]
Update the Terraform AWS GitHub runner to pull in newer features as well as security updates.

When doing this, check if the new version supports instantiating multiple runner pools in one region.

https://github.com/zephyrproject-rtos/terraform-aws-github-runner

**Comment by stephanosio** [comment]
No longer relevant after #43.
